guides/_posts/2014-01-5-vfl.html

```diff
@@ -259,8 +257,8 @@
 
 {% highlight css %}
 /* VFL */
-@horizontal [#box1]~-~[#box2];
-@vertical [#box3]~-~[#box4];
+@h (#box1)~-~(#box2);
+@v (#box3)~-~(#box4);
```

Comment:
Mentioned this before, but I don't think we should encourage using gaps like this without using `gap()`.